package.json

```diff
@@ -15,7 +16,6 @@
     "babel-brunch": "~6.0.0",
     "uglify-js-brunch": "~2.0.1"
   },
-  "files": ["README.md", "LICENSE.md", "package.json", "priv/static/phoenix.js", "web/static/js/phoenix.js"],
```

Comment:
Not really. The above lines (`main` and `jsnext:main`) are all that's needed. This is duplication, actually.

`README` (and it's variants) are already picked up by default (and always have been). `package.json` has to be part of the bundle (per the spec), and the other files are extraneous.